contracts/OgvStaking.sol

```diff
@@ -33,10 +33,11 @@ contract OgvStaking is ERC20Votes {
     mapping(address => uint256) public rewardDebtPerShare;
     uint256 public accRewardPerShare; // As of the start of the block
 
-    // Used to track any manual `delegate()` method call. When this isn't
+    // Used to track any calls to `delegate()` method. When this isn't
     // set to true, voting powers are delegated to the receiver of the stake
-    // when `stake()` or `extend()` method are called.
-    mapping(address => bool) public hasManualDelegation;
+    // when `stake()` or `extend()` method are called. For existing stakers 
+    // with deletegation set, This will always remain `false` (due to upgrades)
```

Comment:
This comment should be updated, since even if existing users call `extend`, `stake`, or `delegate` this value for them should change to true. As demonstrated in this test: https://github.com/OriginProtocol/ousd-governance/pull/377/commits/7287c88635bd6dd1a0d8e26d9378a88d4931b33a

Other than that LGTM

Reply:
Actually, calling `extend` or `stake` won't change the boolean when the user has delegation set already. But `delegate` will switch the flag. Will update the comment